RetsTalk/RetsTalk/Chat/View/RetryView.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  RetryView.swift
+//  RetsTalk
+//
+//  Created by HanSeung on 11/18/24.
+//
+
+import UIKit
+
+final class RetryView: UIView {
+    private let backgroundLabel: UILabel = {
+        let label = UILabel()
+        label.text = Constants.backgroundLabelText
+        label.textColor = .blazingOrange
+        label.font = UIFont.appFont(.body)
+        return label
+    }()
+    
+    private let retryButton: UIButton = {
+        let button = UIButton()
+        button.setTitle(Constants.buttonLabelText, for: .normal)
+        button.setTitleColor(.white, for: .normal)
+        button.backgroundColor = .blazingOrange
+        button.layer.cornerRadius = Metrics.cornerRadius
+        return button
+    }()
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setUp()
```

Comment:
super 다음은 한칸 띄우는 게 좋을 것 같네요!

```suggestion
        super.init(frame: frame)

        setUp()
```